.github/workflows/release.yml

```diff
@@ -10,79 +10,217 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-        arch: [auto, aarch64]
-        py: [cp37, cp38, cp39, cp310]
-        include:
-          - os: macos-latest
-            py: cp38
-            arch: universal2
-          - os: macos-latest
-            py: cp39
-            arch: universal2
-          - os: macos-latest
-            py: cp310
-            arch: universal2
-        exclude:
-          - os: windows-latest
-            arch: aarch64
-          - os: macos-latest
-            arch: aarch64
-    env:
-      CIBW_BUILD: ${{ matrix.py }}-*
-      CIBW_ARCHS_LINUX: ${{ matrix.arch }}
+        os: [ubuntu-latest, windows-latest, macos-12]
+        arch: [native]
+        py: [cp37, cp38, cp39, cp310, cp311]
+        # exclude:
+        #     - os: macos-12
+        #       arch: arm64
+        #       py: cp37
+        #     - os: macos-12
+        #       arch: arm64
+        #       py: cp38
+        # os: [ubuntu-latest, windows-latest, macos-12]
+        # os: [ubuntu-latest, windows-latest]
+        # arch: [native, arm64]
+        # py: [cp37, cp38, cp39, cp310]
     steps:
+
+
       - uses: actions/checkout@v3
         with:
           submodules: true
 
+      # graphviz is needed for tests, only install it when compilation succeed
+      # How to integrate into cibuildwheels?
+      - uses: tlylt/install-graphviz@v1
```

Comment:
Just skip the tests if graphviz cannot be found.

Reply:
Ok